Library/PhantomLib.Droid/PhantomLib.Droid.csproj

```diff
@@ -58,9 +58,6 @@
   <ItemGroup>
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
just wanted to check that removing this was intentional

Reply:
I had to add Xamarin.Forms to this project, which pulled in a metric crap ton of Android support packages with it.